pyproject.toml

```diff
@@ -223,3 +223,6 @@ filterwarnings = [
     "ignore:PY_SSIZE_T_CLEAN will be required.*:DeprecationWarning",
     "ignore:The loop argument is deprecated since Python 3.8.*:DeprecationWarning",
 ]
+
+[tool.repo-review]
+ignore = ["PC111", "PC180"]
```

Comment:
It would be nice to add some comments explaining what the rules being ignored are (and perhaps why they're being ignored?)

Reply:
These are rules for Python code in documentation and for JavaScript.
- I thought I'd leave the documentation as is to start with.
- There only one JavaScript file, in the documentation: `docs/_static/custom.js`.